java/backup/src/main/java/com/instaclustr/backup/downloader/AzureDownloader.java

```diff
@@ -39,7 +39,8 @@ public RemoteObjectReference objectKeyToRemoteReference(final Path objectKey) th
     @Override
     public void downloadFile(final Path localPath, final RemoteObjectReference object) throws Exception {
         final CloudBlockBlob blob = ((AzureRemoteObjectReference) object).blob;
-        Files.createDirectories(localPath);
+        logger.info(String.format("download file with azure remote_path=%s", blob.getUri()));
```

Comment:
I believe the logger supports string substitution via the {} syntax e.g. 
`logger.info("download file with azure remote_path={}", blob.getUri());`

Reply:
yes !